packages/jupyter-ai/package.json

```diff
@@ -78,19 +78,15 @@
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-markdown": "^8.0.6",
-    "react-syntax-highlighter": "^15.5.0",
-    "rehype-katex": "^6.0.2",
-    "remark-math": "^5.1.1"
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@jupyterlab/builder": "^4",
     "@jupyterlab/testutils": "^4",
     "@types/jest": "^29",
-    "@types/react-syntax-highlighter": "^15.5.6",
+    "@types/react-dom": "^18.2.18",
```

Comment:
This should be `"^18.2.0"` to match the version range spec of `react-dom`.

Reply:
Good point